discounts/rust/order-discounts/default/shopify.function.extension.toml.liquid

```diff
@@ -0,0 +1,11 @@
+name = "{{name}}"
+type = "{{extensionType}}"
+version = "2"
+
+[meta_object]
+type = "object"
+
+[meta_object.fields]
+
+[commands]
+build = "make build"
```

Comment:
Looking at the generated Makefile, I think this should be make `build_wasm` or just `make`, no? And the same for the other files.

```suggestion
build = "make"
```